driver-core/Upgrade_guide_to_2.0.rst

```diff
@@ -155,13 +151,28 @@ Other API Changes
     will now need to use Host#isUp and Cluster#register if you were using that
     class.
 
-11. LatencyTracker#update method now has a different signature and takes two new
-    parameters: the statement that has been executed (never null), and the exception
-    thrown while executing the query (or null, if the query executed successfully).
-    Existing implementations of this interface, once upgraded to the new method
-    signature, should continue to work as before.
 
+2.0.10
+------
+
+1. LatencyTracker#update now has a different signature and takes two new
+   parameters: the statement that has been executed (never null), and the exception
+   thrown while executing the query (or null, if the query executed successfully).
+   Existing implementations of this interface, once upgraded to the new method
+   signature, should continue to work as before.
+
+2. SocketOptions#getTcpNoDelay() is now TRUE by default (it was previously undefined).
+   This reflects the new behavior of Netty (which was upgraded from version 3.9.0 to
+   4.0.27): TCP_NODELAY is now turned on by default, instead of depending on the OS
+   default like in previous versions.
+
+3. Netty is not shaded anymore in the default Maven artifact. However we publish a
+   shaded artifact under a different classifier. See `Using the shaded JAR<http://datastax.github.io/java-driver/features/shaded_jar/>`_
```

Comment:
Just a note so that we do not forget to check this URL when the docs are published.